developer-docs-site/docs/concepts/events.md

```diff
@@ -2,8 +2,48 @@
 title: "Events"
 slug: "events"
 ---
+Events are emitted during the execution of a transaction. Each Move module can define its own events and choose when to emit the events upon execution of the module. Aptos move starts with events with handle on mainnet launch, then switch to module events since v1.7.
 
-Events are emitted during the execution of a transaction. Each Move module can define its own events and choose when to emit the events upon execution of the module. For example, during a [coin transfer](../tutorials/first-transaction.md), both the sender and receiver's accounts will emit `SentEvent` and `ReceivedEvent`, respectively. This data is stored within the ledger and can be queried via the REST interface's [Get events by event handle](https://fullnode.devnet.aptoslabs.com/v1/spec#/operations/get_events_by_event_handle).
+# Module Events
+Since v1.7, Aptos move released a new event framework, module events. Every event stream is identified by an event struct type instead of a handle.
```

Comment:
```suggestion
Since v1.7, Aptos Move released a new event framework, module events. Every event stream is identified by an event struct type instead of a handle.
```